app/base/static/property_images/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
+*/
+!.gitignore
```

Comment:
same here 
the gitignore on the base of the project has effect to every file in the project recursively